api/controllers/comments.controller.js

```diff
@@ -29,3 +33,13 @@ exports.postComment = (req, res, next) => {
       next(err);
     });
 };
+
+exports.deleteAComment = (req, res, next) => {
+  const { comment_id } = req.params;
+
+  removeAComment(comment_id)
+    .then(() => {
+      res.status(204).send();
```

Comment:
You can also do `res.sendStatus(204)` - means you dont have to have an empty send.